aytests/sles12_minimal.rb

```diff
@@ -1,5 +1,7 @@
 require "aytests/spec_helper"
 
-describe "SLES 12 checks," do
+describe "SLES 12 minimal," do
+  # udev network rules are not checked because they won't work
+  # in this scenario.
```

Comment:
I know it's not public (for no particular good reason), but what about linking here?
https://wiki.microfocus.net/index.php/YAST#Interface_renaming_problems_with_AutoYaST_in_SLE12-SP1